client/ladder/ladder.tsx

```diff
@@ -620,18 +625,36 @@ export function LadderTable(props: LadderTableProps) {
         />
         <DivisionSelect
           dense={true}
-          label={'Division'}
+          label={t('ladder.divisionLabel', 'Division')}
           value={filteredDivision}
           onChange={onFilteredDivisionChange}
           allowErrors={false}>
-          <SelectOption value={DivisionFilter.All} text={'All'} />
-          <SelectOption value={DivisionFilter.Champion} text={'Champion'} />
-          <SelectOption value={DivisionFilter.Diamond} text={'Diamond'} />
-          <SelectOption value={DivisionFilter.Platinum} text={'Platinum'} />
-          <SelectOption value={DivisionFilter.Gold} text={'Gold'} />
-          <SelectOption value={DivisionFilter.Silver} text={'Silver'} />
-          <SelectOption value={DivisionFilter.Bronze} text={'Bronze'} />
-          <SelectOption value={DivisionFilter.Unrated} text={'Unrated'} />
+          <SelectOption value={DivisionFilter.All} text={t('ladder.division.all', 'All')} />
+          <SelectOption
+            value={DivisionFilter.Champion}
+            text={t('ladder.division.champion', 'Champion')}
```

Comment:
`ladder.division` and `ladder.divisionLabel` seem *very hard* to differentiate.

I'd probably call this one `ladder.divisionGroup` (and the `Label` suffix on the other one still feels weird/unnecessary to me)